Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -460,12 +460,14 @@ namespace AZ
                 cullStats.m_numVisibleCullables += numVisibleCullables;
                 ++cullStats.m_numJobs;
             }
+#else
+            (void)numDrawPackets; // prevent unused variable warning->error
```

Comment:
It does look like `numDrawPackets` is being written in the code above.  Does the unused warning trigger when the variable isn't read from?

Reply:
It is written to on line 354, but never read outside of the `AZ_CULL_DEBUG_ENABLED` block, so it is unused.